### ISSUE TITLE
Cards on home page fail WCAG AA for contrast, Open Sans font isn't loading all weights

### ISSUE THREAD

**Issue by simmscg** [kubeflow/website#1675]
Was looking at #1663, and it alerted me to something else.

- For the card descriptions on the front page, contrast fails WCAG AA at normal text sizes. The background color needs to be darker, the text perhaps lighter.
- The font weights for Open Sans aren't being loaded correctly, so we can't select as many weights as we should be able to.
